src/nifs/sangria/tests.rs

```diff
@@ -166,7 +166,7 @@ where
     const R_P: usize = 3;
 
     let mut f_tr = RelaxedPlonkTrace {
-        U: RelaxedPlonkInstance::new(S.num_challenges, S.round_sizes.len()),
+        U: RelaxedPlonkInstance::new(S.num_challenges, S.round_sizes.len(), S.num_io.len() - 1),
```

Comment:
as previous question, why choose `S.num_io.len() - 1`

Reply:
Because this fn arg is number of step circuit instances, not all instances len